packages/sku/src/services/vite/index.ts

```diff
@@ -7,20 +7,26 @@ import {
   createStartConfig,
 } from './helpers/config/createConfig.js';
 import { cleanTargetDirectory } from '@/utils/buildFileUtils.js';
+import { createOutDir } from './helpers/bundleConfig.js';
+import path from 'node:path';
 import { getAppHosts } from '@/utils/contextUtils/hosts.js';
 import chalk from 'chalk';
 import { prerenderConcurrently } from '@/services/vite/helpers/prerender/prerenderConcurrently.js';
 import allocatePort from '@/utils/allocatePort.js';
 
 export const viteService = {
   build: async (skuContext: SkuContext) => {
+    const outDir = createOutDir(skuContext.paths.target);
     await build(createClientBuildConfig(skuContext));
     await build(createServerBuildConfig(skuContext));
     if (skuContext.routes) {
       await prerenderConcurrently(skuContext);
     }
-    await cleanTargetDirectory(`${process.cwd()}/dist/render`, true);
-    await cleanTargetDirectory(`${process.cwd()}/dist/.vite`, true);
+    await cleanTargetDirectory(outDir.ssg, true);
+    await cleanTargetDirectory(
+      path.join(skuContext.paths.target, '.vite'),
+      true,
+    );
```

Comment:
suggestion: could be nice to encapsulate this logic with the other `createOutDir` stuff. Maybe there can be a little function exported on that object that does the joining for us.


```suggestion
    await cleanTargetDirectory(outDir.join('.vite'), true);
```